ark/phenotyping/cluster.py

```diff
@@ -0,0 +1,151 @@
+import os
+
+import numpy as np
+import pandas as pd
+
+
+def winner(sample, weights):
+    """Find the coordinates to the winning neuron in the SOM
+
+    Args:
+        sample (numpy.ndarray):
+            A row in the pixel matrix identifying information associated with one pixel
+        weights (numpy.ndarray):
+            A square matrix identifying the weights between two neurons
+
+    Returns:
+        tuple:
+            A coordinate array which indicates the winning neuron's position
```

Comment:
Simpler would be "the coordinates of the winning neuron"

Reply:
I can do better: a coordinate defined in the x-y plane, which should contain two values, both of which should be contained in the set of whole numbers, which defines the position of the neuron in the weights matrix which corresponds to the closest Euclidean distance to the weights in question 😝.